lib/check_statistics.rb

```diff
@@ -16,7 +16,7 @@ def requests(team_id, platform, start_date, end_date, language, type = nil)
     def reports_received(team_id, platform, start_date, end_date, language)
       TiplineRequest
         .where(team_id: team_id, language: language, created_at: start_date..end_date, platform: platform)
-        .where('smooch_report_received_at IS NOT NULL')
+        .where.not(smooch_report_received_at: 0)
```

Comment:
I think it can still be `NULL`, no? I think it's more robust to change this condition to:
```
.where('smooch_report_received_at > 0')
```

Reply:
No as I set the default value to `0` in migration file so it'll be a value or `0`